<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="1" ignored="1" total="6" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-03-03T10:22:12 IST" name="Suite" finished-at="2025-03-03T10:23:16 IST" duration-ms="63614">
    <groups>
      <group name="ErrorHandling">
        <method signature="ErrorValidation.loginErrorValidation()[pri:0, instance:Seleniumlearning.Manasaproject.ErrorValidation@3576ddc2]" name="loginErrorValidation" class="Seleniumlearning.Manasaproject.ErrorValidation"/>
      </group> <!-- ErrorHandling -->
      <group name="Purchase">
        <method signature="ProjectTestNG.submitOrder(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:Seleniumlearning.Manasaproject.ProjectTestNG@24c1b2d2]" name="submitOrder" class="Seleniumlearning.Manasaproject.ProjectTestNG"/>
      </group> <!-- Purchase -->
    </groups>
    <test started-at="2025-03-03T10:22:12 IST" name="ProjectTestNG" finished-at="2025-03-03T10:22:44 IST" duration-ms="31486">
      <class name="Seleniumlearning.Manasaproject.ProjectTestNG">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:Seleniumlearning.Manasaproject.ProjectTestNG@24c1b2d2]" started-at="2025-03-03T10:22:12 IST" name="launchApplication" finished-at="2025-03-03T10:22:17 IST" duration-ms="4497" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:Seleniumlearning.Manasaproject.ProjectTestNG@24c1b2d2]" started-at="2025-03-03T10:22:17 IST" name="submitOrder" data-provider="getData" finished-at="2025-03-03T10:22:28 IST" duration-ms="11534" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[rahulshettyacademy]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[learning]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Blackberry]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Seleniumlearning.Manasaproject.ProjectTestNG@24c1b2d2]" started-at="2025-03-03T10:22:28 IST" name="tearDown" finished-at="2025-03-03T10:22:29 IST" duration-ms="450" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:Seleniumlearning.Manasaproject.ProjectTestNG@24c1b2d2]" started-at="2025-03-03T10:22:29 IST" name="launchApplication" finished-at="2025-03-03T10:22:32 IST" duration-ms="3405" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:Seleniumlearning.Manasaproject.ProjectTestNG@24c1b2d2]" started-at="2025-03-03T10:22:32 IST" name="submitOrder" data-provider="getData" finished-at="2025-03-03T10:22:43 IST" duration-ms="11152" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[rahulshettyacademy]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[learning]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[iphone X]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Seleniumlearning.Manasaproject.ProjectTestNG@24c1b2d2]" started-at="2025-03-03T10:22:43 IST" name="tearDown" finished-at="2025-03-03T10:22:44 IST" duration-ms="292" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Seleniumlearning.Manasaproject.ProjectTestNG -->
    </test> <!-- ProjectTestNG -->
    <test started-at="2025-03-03T10:22:45 IST" name="ErrorValidation" finished-at="2025-03-03T10:23:16 IST" duration-ms="31128">
      <class name="Seleniumlearning.Manasaproject.ErrorValidation">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:Seleniumlearning.Manasaproject.ErrorValidation@3576ddc2]" started-at="2025-03-03T10:22:45 IST" name="launchApplication" finished-at="2025-03-03T10:22:48 IST" duration-ms="3229" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="ProductErrorValidation()[pri:0, instance:Seleniumlearning.Manasaproject.ErrorValidation@3576ddc2]" started-at="2025-03-03T10:22:48 IST" name="ProductErrorValidation" finished-at="2025-03-03T10:22:59 IST" duration-ms="10575" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ProductErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Seleniumlearning.Manasaproject.ErrorValidation@3576ddc2]" started-at="2025-03-03T10:22:59 IST" name="tearDown" finished-at="2025-03-03T10:22:59 IST" duration-ms="277" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:Seleniumlearning.Manasaproject.ErrorValidation@3576ddc2]" started-at="2025-03-03T10:22:59 IST" name="launchApplication" finished-at="2025-03-03T10:23:02 IST" duration-ms="3468" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method retried="true" signature="loginErrorValidation()[pri:0, instance:Seleniumlearning.Manasaproject.ErrorValidation@3576ddc2]" started-at="2025-03-03T10:23:02 IST" name="loginErrorValidation" finished-at="2025-03-03T10:23:07 IST" duration-ms="4409" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect username/password.] but found [Incorrect usernam/password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect username/password.] but found [Incorrect usernam/password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at Seleniumlearning.Manasaproject.ErrorValidation.loginErrorValidation(ErrorValidation.java:33)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Seleniumlearning.Manasaproject.ErrorValidation@3576ddc2]" started-at="2025-03-03T10:23:07 IST" name="tearDown" finished-at="2025-03-03T10:23:07 IST" duration-ms="379" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:Seleniumlearning.Manasaproject.ErrorValidation@3576ddc2]" started-at="2025-03-03T10:23:07 IST" name="launchApplication" finished-at="2025-03-03T10:23:10 IST" duration-ms="3184" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="loginErrorValidation()[pri:0, instance:Seleniumlearning.Manasaproject.ErrorValidation@3576ddc2]" started-at="2025-03-03T10:23:10 IST" name="loginErrorValidation" finished-at="2025-03-03T10:23:15 IST" duration-ms="4679" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect username/password.] but found [Incorrect usernam/password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect username/password.] but found [Incorrect usernam/password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at Seleniumlearning.Manasaproject.ErrorValidation.loginErrorValidation(ErrorValidation.java:33)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Seleniumlearning.Manasaproject.ErrorValidation@3576ddc2]" started-at="2025-03-03T10:23:16 IST" name="tearDown" finished-at="2025-03-03T10:23:16 IST" duration-ms="251" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Seleniumlearning.Manasaproject.ErrorValidation -->
    </test> <!-- ErrorValidation -->
  </suite> <!-- Suite -->
</testng-results>
